yarn.lock

```diff
@@ -16824,7 +16829,7 @@ webpack-virtual-modules@^0.6.1:
   resolved "https://registry.yarnpkg.com/webpack-virtual-modules/-/webpack-virtual-modules-0.6.2.tgz#057faa9065c8acf48f24cb57ac0e77739ab9a7e8"
   integrity sha512-66/V2i5hQanC51vBQKPH4aI8NMAcBW59FVBs+rC7eGHupMyfn34q7rZIE+ETlJ+XTevqfUhVVBgSUNSW2flEUQ==
 
-webpack@5, webpack@^5.37.1, webpack@^5.88.2:
+webpack@5, webpack@^5.88.2:
```

Comment:
Seems like all webpack dependencies could be merged together. My recomendation is delete this one and the bottom one, and run yarn install, it should create the same dependency for all the versions